### DISCUSSION TITLE
Intermittent crackle/distortion and/or latency in Ableton

### DISCUSSION THREAD

**Discussion by i-am-dan** [ExistentialAudio/BlackHole#240] [General]
Hi! First of all, love the app. thank you.

The audio sometimes starts to crackle and latency randomly gets introduced. It seems like changing the sample size fixes this every time I get some audio distortion. So, lets say I have the buffer to be 512. I get some latency, I change to 128/256 etc. Latency goes away. Audio distorts, I change to 128/64/512... etc. The distortion goes away. 

I'm on Mojave.

Any thoughts on this would be VERY helpful! TY!

**Comment by devinroth** [comment]
Sounds like you don’t have drift correction enabled for BlackHole.

Devin

**Reply by jakemasca** [reply]
Hi Devin and Dan,

I have the exact same issue. I also have drift correction for black hole.

The issue happens only with Ableton for me. I don’t have any issues with this same setup on Logic Pro.

The latency introduced in ableton comes in randomly. Not sure why.

**Comment by devinroth** [comment]
So no problems in Logic but issues on Ableton.

What are you using as the input / output of live? And buffer size?

Devin
